### PR TITLE
Add version recommendation, quota notes and test details to TURN server documentation

### PR DESCRIPTION
**Pull request by danxuliu**

### CODE REVIEW

File: docs/TURN.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ The configuration of Nextcloud Talk mainly depends on your desired usage:
 
 ## Install and setup _coTURN_ as TURN server
 
+It is recommended to install _coTURN_ >= 4.5.0.8. In previous versions there is a bug that causes [the IPv6 UDP sockets created by coTURN not to be freed](https://github.com/coturn/coturn/issues/217). Due to this the _turn_ process ends not being able to open new ports and thus not being able to serve new connections. Moreover, when that happens, even if there are no connections a high CPU load will be caused by the _turn_ process. Therefore, if you can not install _coTURN_ >= 4.5.0.8 you should restart the _turn_ process periodically to work around that issue.
```
Comment (nickvergessen) [inline docs/TURN.md:14]: It's It is recommended to install the newest version, at least 4.5.0.8 ?

Comment (danxuliu) [reply docs/TURN.md:14]: At least 4.5.0.8, as the linked bug was fixed in that version.

**Review by Ivansss — APPROVED** [review]
👍

**Review by nickvergessen — APPROVED** [review]
Fix up needed :roll_eyes:

**Pull request merged**